bldg61/bldg61WebApp
# Two calendars

### How does this get shipped to prod?

Ship and check the node version (had some node version issues with connecting to the db

### What does this PR do?

Bumps some dependency issues, adds the new virtual calendar to the root view

### Screenshots or gifs of the new hotness

<img width="1052" alt="Screen Shot 2020-08-04 at 3 36 59 PM" src="https://user-images.githubusercontent.com/6353499/89347665-7076d400-d668-11ea-8f40-c1fd4788a051.png">


### How do I verify?

Make sure you can see both calendars, and that after logging in, checkouts can be seen.

### Any new dependencies?

Pinned node to avoid database weirdness (that's going to be fun to bump).

### gif

![](https://media.giphy.com/media/aWNByu8u6sati/giphy.gif)
